recipe/post-link.bat

```diff
@@ -0,0 +1,2 @@
+python %PREFIX%\bin\cudatoolkit-dev-post-install.py
+if errorlevel 1 exit 1
```

Comment:
it is a good practice to have an empty line at the end of the file. In general, you can configure your IDE to do it for you automatically. but don't need to rebuild here to change that now.

Reply:
Ok, thanks. Will incorporate this tip.